dev/bots/travis_script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -ex
+
+export PATH="$PWD/bin:$PWD/bin/cache/dart-sdk/bin:$PATH"
+
+if [ "$SHARD" != "build_and_deploy_gallery" ]; then
```

Comment:
this would probably be clearer if we flipped the if and else clauses around

Reply:
Done